components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -90,9 +116,123 @@ const WorkspacesPage: FunctionComponent = () => {
         } catch (e) {}
     }, [deleteInactiveWorkspaces, inactiveWorkspaces, toast]);
 
+    const [showGettingStarted, setShowGettingStarted] = useState(true);
+    const dismissGettingStarted = useCallback(() => {
+        setShowGettingStarted(false);
+
+        mutateUser(
+            {
+                additionalData: {
+                    profile: {
+                        coachmarksDismissals: {
+                            [GETTING_STARTED_DISMISSAL_KEY]: new Date().toISOString(),
+                        },
+                    },
+                },
+            },
+            {
+                onError: (e) => {
+                    toast("Failed to dismiss getting started");
+                    setShowGettingStarted(true);
+                },
+            },
+        );
+    }, [mutateUser, toast]);
+
+    const [isVideoModalVisible, setVideoModalVisible] = useState(false);
+    const handleVideoModalClose = useCallback(() => {
+        setVideoModalVisible(false);
+    }, []);
+
     return (
         <>
-            <Header title="Workspaces" subtitle="Manage recent and stopped workspaces." />
+            <Header
+                title="Workspaces"
+                subtitle="Manage, start and stop your personal development environments in the cloud."
```

Comment:
Copy comes from [this Figma](https://www.figma.com/design/Qm7uJFwCvffLpHHAzFeZOo/Classic-Improve-Onboarding?node-id=142-6089&m=dev).